portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/dataset/DataSetViewFields.testcase

```diff
@@ -1481,6 +1481,320 @@ definition {
+
+		task ("And a new view") {
+			DataSetAdmin.createFDSViewViaAPI(
+				dataSetName = "Data Set Test",
+				key_dataSetViewNameList = "View Test");
+		}
+
+		task ("When the user goes to the Filter tab") {
+			DataSetAdmin.goToSpecificViewPage(
+				dataSetName = "Data Set Test",
+				key_viewName = "View Test");
+
+			DataSetAdmin.goToTab(tabName = "Filters");
+		}
+
+		task ("And creates a new client extension filter") {
+			LexiconEntry.gotoAdd();
+
+			MenuItem.click(menuItem = "Client Extension");
+
+			Type(
+				key_text = "Name",
+				locator1 = "TextInput#ANY",
+				value1 = "Liferay Sample Data Set Filter");
+
+			Click(
+				key_indexNumber = 1,
+				key_text = "Select",
+				locator1 = "Button#ANY_N");
+
+			MenuItem.click(menuItem = "name");
+
+			Click(
+				key_indexNumber = 1,
+				key_text = "Select",
+				locator1 = "Button#ANY_N");
+
+			MenuItem.click(menuItem = "Client Extension Filter Test");
+		}
+
+		task ("And saves the new filter") {
+			PortletEntry.save();
+		}
+	}
+
+	@description = "LPS-187522. Allow administrators to configure client extension filters on dataset views."
+	@priority = 5
+	test ClientExtensionFrontendDataSetFilterCreationValidationTestwithEmptyName {
+		task ("Given a new Client Extension Frontend Data Set filter") {
+			ClientExtensionGeneral.goToRemoteAppsPortlet();
+
+			ClientExtensionGeneral.addFrontendDataSetFilter(
+				description = "New Client Extension Frontend Data Set Filter",
+				javaScriptURL = "https://liferay.github.io/liferay-frontend-projects/date-filter.js",
+				name = "Client Extension Filter Test");
+		}
+
+		task ("And a new data set entry") {
+			DataSetAdmin.goToDataSetAdminPage();
+
+			DataSetAdmin.createDataSetEntryViaAPI(
+				dataSetName = "Data Set Test",
+				restApplication = "/data-set-manager/fields",
+				restEndpoint = "/",
+				restSchema = "FDSField");
+		}
+
+		task ("And a new view") {
+			DataSetAdmin.createFDSViewViaAPI(
+				dataSetName = "Data Set Test",
+				key_dataSetViewNameList = "View Test");
+		}
+
+		task ("When the user goes to the Filter tab") {
+			DataSetAdmin.goToSpecificViewPage(
+				dataSetName = "Data Set Test",
+				key_viewName = "View Test");
+
+			DataSetAdmin.goToTab(tabName = "Filters");
+		}
+
+		task ("And creates a new client extension filter") {
+			LexiconEntry.gotoAdd();
+
+			MenuItem.click(menuItem = "Client Extension");
+
+			Type(
+				key_text = "Name",
+				locator1 = "TextInput#ANY",
+				value1 = "");
+
+			Click(
+				key_indexNumber = 1,
+				key_text = "Select",
+				locator1 = "Button#ANY_N");
+
+			MenuItem.click(menuItem = "name");
+
+			Click(
+				key_indexNumber = 1,
+				key_text = "Select",
+				locator1 = "Button#ANY_N");
+
+			MenuItem.click(menuItem = "");
+		}
+
+		task ("And saves the new filter") {
+			PortletEntry.save();
+		}
+	}
+
+	@description = "LPS-187522. Allow administrators to configure client extension filters on dataset views."
+	@priority = 5
+	test ClientExtensionTestinDataSetManager {
```

Comment:
This test (Im just guessing, because you didn't put enough information what are you testing) it is supposed to test that, when there are no client extension created, the user cannot create a filter bases on a client extensions. So (if this is correct), the test name should be something like "ClientExtensionFilterCannotBeCreatedWithoutClientExtension"